### ISSUE TITLE
issue while detecting vertical lines using pre-trained model

### ISSUE THREAD

**Issue by trohit920** [Hanqer/deep-hough-transform#54]
Hello, 
I have tried the DHT pre-trained models to detect vertical catenary  lines but the accuracy is not great. Please check the attached image files. How do you think i should solve this issue, shall i retrain the model on my custom data?
Any suggestion will be of great help.
![Screenshot from 2023-04-05 11-05-55](https://user-images.githubusercontent.com/30137687/229962582-55f82cc2-e3a1-48c6-8ab1-3f08c40e78f7.png)

![Screenshot from 2023-04-05 11-03-59](https://user-images.githubusercontent.com/30137687/229962572-f3044fed-2399-4d39-8ffa-4baf6dfaf3a0.png)

**Comment by zeakey** [comment]
I guess the model is highly biased towards horizontal lines because of the training data. You may alleviate this by diversifying your training samples with data augmentation, e.g. rotating.

Sorry for the late response.
